server.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const mongoose = require("mongoose");
+
+mongoose.connect("mongodb://localhost/LoginDB").then(() => {
+  console.log("Connected to the database");
+});
+
+const registerRouter = require("./registerRouter");
+
+const server = express();
+
+server.use(express.json());
+server.use("/api", registerRouter);
```

Comment:
Remember that using just `/api` here will prevent you from writing other Routers that use `/api/...` Have you considered something like `server.use("/register", registerRouter);`?